src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -19,7 +19,7 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameters: INDEX (must be a positive integer, greater than or equals to 1)\n"
```

Comment:
Would it be possible to include this change for the other index commands as well?

Reply:
Okie, will add this change to edit, mail index and deltask commands